src/services/http.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+const http = axios.create();
+
+http.defaults.baseURL = process.env.BASE_URL;
+http.defaults.headers = {
+  // TODO: change to correct
+  'Access-Control-Allow-Origin': '*',
```

Comment:
This is supposed to be handled on the responding side not from the requesting party. Do I miss something?